src/fasta-recs.py

```diff
@@ -11,8 +10,22 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
+    # print(f"Now I need to process the records in {args.fasta}")
+
+    printer = ''
+    for line in (args.fasta):
+        if '>' in line:
```

Comment:
There is a potential problem here. You are checking if the character `>` appears anywhere in `line`, not if it appears at the beginning. This is unlikely to give you any problems, and certainly will not in this class, but it isn't quite what you want. You would be better served with `line.startswith('>')`.